packages/cli-config/src/findDependencies.ts

```diff
@@ -1,5 +1,57 @@
 import path from 'path';
 import fs from 'fs';
+import glob from 'glob';
+import findRoot from 'find-root';
+
+/**
+ * Returns an array of dependencies from workspaces's package.json
+ */
+function findWorkspacesDependencies(from: string): Array<string> {
+  // find workspaces root if any
+  let root: string | undefined;
+  try {
+    root = findRoot(from, (dir) => {
+      const pjsonPath = path.join(dir, 'package.json');
+      return (
+        fs.existsSync(pjsonPath) &&
+        typeof require(pjsonPath).workspaces !== 'undefined'
+      );
+    });
+  } catch (e) {
+    // noop.
+  }
+
+  // workspaces root is not defined
+  if (typeof root === 'undefined') {
+    return [];
+  }
+
+  const rootPjson: {
+    workspaces: Array<string> | {packages: Array<string>};
+  } = require(path.join(root, 'package.json'));
+  let dependencies: Array<string> = [];
+  const workspaces = Array.isArray(rootPjson.workspaces)
+    ? rootPjson.workspaces
+    : rootPjson.workspaces.packages;
```

Comment:
I'm not a fan of relying on the `workspaces`. There are monorepos that don't use this kind of resolution strategy. Wonder what are @grabbou thoughts on that